io.ts

```diff
@@ -0,0 +1,109 @@
+export const fileObjectDefinition : string = 
+`class File(object):
+    fd : int = 0          
+    mode : int = 0
+    closed : bool = False
+    pointer : int = 0
+    filelength : int = 0
+
+    def __init__(self : File):
+        pass
+    
+    def read(self : File) -> int:
+        if self.mode != 0:
+            print(44444)
```

Comment:
Maybe replace these prints with exceptions?

Reply:
Do you only read one byte always? Is it the same byte?

Reply:
We have replaced the prints with the JS exceptions.
The read also moves the file pointer, so we would be reading the bytes in order.